src/miral/external_client.cpp

```diff
@@ -71,31 +73,25 @@ void miral::ExternalClientLauncher::operator()(mir::Server& server)
     self->server = &server;
 
     static auto const app_env = "app-env";
-    static auto const default_env = "GDK_BACKEND=wayland,x11:QT_QPA_PLATFORM=wayland:SDL_VIDEODRIVER=wayland:"
+    static auto const default_env =
+        "GDK_BACKEND=wayland,x11:QT_QPA_PLATFORM=wayland:SDL_VIDEODRIVER=wayland:"
         "-QT_QPA_PLATFORMTHEME:NO_AT_BRIDGE=1:QT_ACCESSIBILITY:QT_LINUX_ACCESSIBILITY_ALWAYS_ON:"
         "_JAVA_AWT_WM_NONREPARENTING=1:-GTK_MODULES:-OOO_FORCE_DESKTOP:-GNOME_ACCESSIBILITY:";
     static auto const app_x11_env = "app-env-x11";
     static auto const default_x11_env = "GDK_BACKEND=x11:QT_QPA_PLATFORM=xcb:SDL_VIDEODRIVER=x11";
 
-    server.add_configuration_option(
-        app_env,
-        "Environment for launched apps",
-        default_env);
+    server.add_configuration_option(app_env, "Environment for launched apps", default_env);
 
-    server.add_configuration_option(
-        app_x11_env,
-        "X11 changes to --app-env for launched apps",
-        default_x11_env);
+    server.add_configuration_option(app_x11_env, "X11 changes to --app-env for launched apps", default_x11_env);
 
-    server.add_init_callback([self=self, &server]
-         {
-             auto const options = server.get_options();
+    server.add_init_callback([self = self, &server] {
+        auto const options = server.get_options();
```

Comment:
There seems to be no option to place a newline _before_ the opening brace of a lambda to be consistent with the rest of our brace placement.